package.json

```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "cldr-data": ">=25",
     "globalize": "^1.1.0-rc.5 <1.3.0",
-    "webpack": "^2.5.1"
+    "webpack": "^3.0.0"
```

Comment:
>> I thought the trend was to have master being the latest and greatest and patches backported to "old" branches ?

> It could be as you suggest actually... @jbellenger, any comments?

I think a major bump on gwp makes sense here.
Because gwp is so coupled to webpack, I'd be onboard with bumping gwp and rgwp to 3.x to make version compatibility clearer.